core/shared/src/test/scala/fs2/ChunkSpec.scala

```diff
@@ -199,6 +199,11 @@ class ChunkSpec extends Fs2Spec {
     assert(arr2 === Array(0, 0))
   }
 
+  "zipWithIndex andThen toArray" in {
+    val chunk = fs2.Chunk("x", "y", "z")
+    assert(chunk.zipWithIndex.toArray === chunk.toArray.zipWithIndex)
```

Comment:
Can you make this a property based test? :)

Reply:
Oops I read above as "proper test", not property test 🤣